sematic/ui/src/components/SideBar.tsx

```diff
@@ -47,10 +47,20 @@ export default function SideBar() {
             sx={{ color: "rgba(255, 255, 255, 0.5)" }}
             underline="none"
           >
-            <BatchPrediction fontSize="large" />
+            <Timeline fontSize="large" />
             <Typography fontSize={10}>Pipelines</Typography>
           </Link>
         </Box>
+        <Box mt={5}>
+          <Link
```

Comment:
I am working towards gradually shifting from inline [JSS](https://cssinjs.org/?v=v10.9.2) to [styled component](https://mui.com/material-ui/customization/how-to-customize/#2-reusable-component). So we separate markups (structure) concerns from style concerns. I hope you can help join this effort too.